db/migrate/20170307170001_add_turn_to_games.rb

```diff
@@ -0,0 +1,5 @@
+class AddTurnToGames < ActiveRecord::Migration[5.0]
+  def change
+    add_column :games, :turn, :integer
```

Comment:
If you do something like `active_player_id`

Then you could use an association 'belongs_to :active_player, class_name: 'User'`

Then `game.active_player` would return the user for you.